checks/escaping.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * Checks for common escaping issues.
+ */
+
+class EscapingCheck implements themecheck {
+	protected $error = array();
+
+	function check( $php_files, $css_files, $other_files ) {
+
+		$ret = true;
+
+		$checks = array(
+			'/echo get_theme_mod/'              => __( 'Theme options must be escaped.', 'theme-check' ),
+			'/="<\?php esc_html_e/'             => __( 'Use esc_attr_e() inside HTML attributes.', 'theme-check' ),
+			'/="<\?php echo esc_html__/'        => __( 'Use esc_attr__() inside HTML attributes.', 'theme-check' ),
```

Comment:
How do you distinguish when this needs to be triggered? It's not that easy to check attributes. For instance, `href` is an attribute, but if it's populated via variable it should be escaped using `esc_url`. Also, these are for translatable escaping. 

I'm just worried this could trigger false positives 🤷‍♂️

Reply:
I share the concern but I also think the simpler the better...